docs/resources/Channel.md

```diff
@@ -498,6 +498,107 @@ Embed types are "loosely defined" and, for the most part, are not used by our cl
 | type     | integer   | the [type of channel](#DOCS_RESOURCES_CHANNEL/channel-object-channel-types) |
 | name     | string    | the name of the channel                                                     |
 
+### Allowed Mentions Object
+
+The allowed mention field allows for more granular control over mentions without various hacks to the message content. This will always validate against message content to avoid phantom pings (e.g. to ping everyone, you must still have `@everyone` in the message content), and check against user/bot permissions.
+
+###### Allowed Mention Types
+
+| Type              | Value      | Description                           |
+| ----------------- | ---------- | ------------------------------------- |
+| Role Mentions     | "roles"    | Controls role mentions                |
+| User Mentions     | "users"    | Controls user mentions                |
+| Everyone Mentions | "everyone" | Controls @everyone and @here mentions |
+
+###### Allowed Mentions Structure
+
+| Field | Type                           | Description                                                                                                   |
+| ----- | ------------------------------ | ------------------------------------------------------------------------------------------------------------- |
+| parse | array of allowed mention types | An array of [allowed mention types](#DOCS_RESOURCES_CHANNEL/allowed-mention-types) to parse from the content. |
+| roles | list of snowflakes             | Array of role_ids to mention (Max size of 100)                                                                |
+| users | list of snowflakes             | Array of user_ids to mention (Max size of 100)                                                                |
```

Comment:
Personal opinion here, probably subject to discussion, (and I recognize that this PR is already merged) but I think the `Allowed Mention Types` section should go *below* the `Allowed Mentions Structure` section, so that the user knows what the `Allowed Mention Types` section is referring to by the time they get to it. Currently, they read that section and aren't sure what to make of it, until they get to the `Allowed Mentions Structure` section.

It also just makes it more consistent with other parts of the documentation, like the [User Object section](https://discordapp.com/developers/docs/resources/user#user-object), where the tables describing enum-like things come *after* the tables that refer to them.